src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -199,12 +200,26 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
     }
 
     updateRetroFeedback(params: any) {
+        let assigneeIDPrevValue: any;
         if (params.colDef.field === 'AssigneeID' && !params.newValue) {
             params.newValue = 0;
         }
         const updatedRetroFeedbackData = {
             [params.colDef.field]: params.newValue
         };
+        if (params.colDef.field === 'Scope') {
+            assigneeIDPrevValue = params.data.AssigneeID;
+            if (params.newValue === RETRO_FEEDBACK_SCOPE_TYPES.Team) {
+                params.data.Scope = RETRO_FEEDBACK_SCOPE_TYPES.Team;
+                params.data.AssigneeID = null;
+                updatedRetroFeedbackData['AssigneeID'] = 0;
```

Comment:
Since for **team**, it would always be null, we can revert back to our previous logic and stop sending a `0` in `AssigneeID` field.

Reply:
done